cli/pcluster/configure/easyconfig.py

```diff
@@ -258,18 +281,38 @@ def _ask_for_subnets(subnet_list, vpc_section):
     return vpc_parameters
 
 
-def _choose_network_configuration(scheduler):
+def _choose_network_configuration(scheduler, qualified_azs):
     if scheduler == "awsbatch":
         return PublicPrivateNetworkConfig()
+    common_availability_zones = set(qualified_azs["azs_for_master_type"]) & set(qualified_azs["azs_for_compute_type"])
+
+    if not common_availability_zones:
+        print(
+            "Error: There is no single availability zone offering master and compute in current region.\n",
```

Comment:
Nice-to-have: Let's try to make this error message more clear for the user.

The points we want user to know:
* There is no single availability zone offering master and compute instance type in current region
* `pcluster configure` network automation currently only support creating subnets in the same availability zone
* To get around this issue, user need to not use network automation and specify appropriate subnets in pcluster configuration file manually

Let's see how we can phrase these into our error message